frontend/src/pages/AccountSend.vue

```diff
@@ -520,18 +573,63 @@ function useSendForm() {
     if (!token.value?.address) throw new Error(tc('Send.select-a-token'));
     if (!recipientId.value) throw new Error(tc('Send.enter-a-recipient'));
 
+    sendMax.value = true;
+
     if (NATIVE_TOKEN.value?.address === token.value?.address) {
       if (!userAddress.value || !provider.value) throw new Error(tc('Send.wallet-not-connected'));
       const fromAddress = userAddress.value;
       const recipientAddress = await toAddress(recipientId.value, provider.value);
-      const { ethToSend } = await umbraUtils.getEthSweepGasInfo(fromAddress, recipientAddress, provider.value);
-      humanAmount.value = formatUnits(ethToSend, token.value.decimals);
-      return ethToSend;
+      const { ethToSend } = await umbraUtils.getEthSweepGasInfo(
+        fromAddress,
+        recipientAddress,
+        provider.value,
+        {
+          // We override the gasLimit here because we are sending to an
+          // address that has never been seen before, which increases gas
+          // costs and is not accounted for by getEthSweepGasInfo.
+          gasLimit: await estimateNativeSendGasLimit(),
+        }
+      );
+      humanAmount.value = formatUnits(ethToSend.sub(toll.value), token.value.decimals);
+    } else {
+      const tokenBalance = balances.value[token.value.address];
+      humanAmount.value = formatUnits(tokenBalance.toString(), token.value.decimals);
+    }
+  }
+
+  // Get an accurate estimate of the amount of gas needed to perform a native send.
+  async function estimateNativeSendGasLimit() {
+    // Increase estimate to give us some wiggle room if network conditions are volatile.
+    // These have each been configured on a case-by-case basis; do not change
+    // unless you have tested a native token Umbra send on the network with the new value.
```

Comment:
How did we come up with the current scale factors / what's the process for testing here? A single send or two isn't really sufficient since maybe you tested when gas prices were stable. I feel like we're more eyeballing these, which is fine for now

Reply:
Yeah, just eyeballing

Reply:
This is no longer necessary and the code has been removed. Resolving